### PR TITLE
Update utils.py

### PR DESCRIPTION
**Pull request by jonathanng**
Fixed bug

### CODE REVIEW

**Comment by twiecki** [comment]
Thanks -- seems like tz issues creeping up:
```
======================================================================
ERROR: test_create_bayesian_tear_sheet_breakdown_1 (pyfolio.tests.test_tears.PositionsTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/miniconda/envs/testenv/lib/python2.7/site-packages/nose_parameterized/parameterized.py", line 365, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "/home/travis/miniconda/envs/testenv/lib/python2.7/site-packages/matplotlib/testing/decorators.py", line 118, in wrapped_function
    func(*args, **kwargs)
  File "/home/travis/build/quantopian/pyfolio/pyfolio/tests/test_tears.py", line 138, in test_create_bayesian_tear_sheet_breakdown
    **kwargs)
  File "/home/travis/build/quantopian/pyfolio/pyfolio/plotting.py", line 56, in call_w_context
    return func(*args, **kwargs)
  File "/home/travis/build/quantopian/pyfolio/pyfolio/tears.py", line 863, in create_bayesian_tear_sheet
    end=returns.index[-1]))
  File "/home/travis/build/quantopian/pyfolio/pyfolio/utils.py", line 490, in get_symbol_rets
    end=end)
  File "/home/travis/build/quantopian/pyfolio/pyfolio/utils.py", line 265, in default_returns_func
    end=datetime.now())
  File "/home/travis/build/quantopian/pyfolio/pyfolio/utils.py", line 168, in get_returns_cached
    if pd.Timestamp(mtime, unit='s') < latest_dt:
  File "pandas/tslib.pyx", line 1041, in pandas.tslib._Timestamp.__richcmp__ (pandas/tslib.c:19696)
  File "pandas/tslib.pyx", line 1096, in pandas.tslib._Timestamp._assert_tzawareness_compat (pandas/tslib.c:20630)
TypeError: Cannot compare tz-naive and tz-aware timestamps
```

**Comment by jonathanng** [comment]
Ok I fixed it.

**Comment by twiecki** [comment]
After fixing pep8 issues:
```
pyfolio/utils.py:162:21: F821 undefined name 'pytz'
pyfolio/utils.py:163:1: W293 blank line contains whitespace
```

This should be good to go.

**Comment by twiecki** [comment]
I remember that we had it like that originally though, so likely someone changed it.

**Comment by jonathanng** [comment]
Done!

**Comment by gusgordon** [comment]
Thanks for helping out - looks like there's still that error popping up in tests: https://travis-ci.org/quantopian/pyfolio/jobs/179380738

`Cannot compare tz-naive and tz-aware timestamps`

**Comment by twiecki** [comment]
Seems like tests are still failing.

**Comment by jonathanng** [comment]
Ok...let's try this one more time. Sorry about the delay.

File: pyfolio/utils.py [hunk 1]
```diff
@@ -165,7 +165,13 @@ def get_returns_cached(filepath, update_func, latest_dt, **kwargs):
             raise
         update_cache = True
     else:
-        if pd.Timestamp(mtime, unit='s') < _1_bday_ago():
```
Comment (twiecki) [inline pyfolio/utils.py]: can `_1_bday_ago` be deprecated then?

Comment (jonathanng) [inline pyfolio/utils.py]: It doesn't appear so. It's still being used here:

- `default_returns_fun`
- `load_portfolio_risk_factors`
- `get_treasury_yield`

**Comment by gusgordon** [comment]
@jonathanng Merging, thanks for contributing!

**Pull request merged**